src/main/java/org/mamute/controllers/AuthController.java

```diff
@@ -36,6 +36,7 @@
 	@Get
 	public void loginForm(String redirectUrl) {
 		if (loggedUser.isLoggedIn()) {
+			result.include("loginRequiredMessages", Collections.emptyList());
```

Comment:
This is required in the special case where LDAP SSO is used:
* user opens for example the "new question" page (while not logged in yet)
* user is redirected to the login page
* LDAP SSO is logging the user in automatically (note: it's best practice to enable LDAP SSO only on the login page, especially with Microsoft AD servers)
* user is redirected back to the "new question" page
* OLD behavior: "you need to login" message was shown on the "new question" page, new behavior: message is gone (because it was cleared in AuthController)

If you want I can move this to the LDAP SSO filter, but that would mean we'd have to re-implement it when we have another "automatic login method".